docs/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 * Weapons CSV download now includes a Zoom stat column.
 * Shaders, ornaments, and mods can now be searched in their choosers.
+* Changed the Loadout Optimizer's Armor Upgrade options for Assume Masterwork and Lock Element options. All armor will now have an assumed minimum energy capacity of 7. The new settings have the following options,
+  * Assumed Masterwork
```

Comment:
Would it help to have longer tooltips for these options?

Reply:
Yeah that is probably a good idea, I will add it tonight if I get a chance